configuration/persistence.md

```diff
@@ -76,6 +90,8 @@ where `<itemlist>` is a comma-separated list consisting of one or more of the fo
 - `<itemName>` a single Item identified by its name. This Item can be a group Item.  But note that only the group value will be persisted.  The value of the individual group members will not be persisted using this option.
 - `<groupName>*` - all members of this group will be persisted, but not the group itself. If no strategies are provided, the default strategies that are declared in the first section are applied.  Optionally, an alias may be provided if the persistence service requires special names (e.g. a table to be used in a database, a feed id for an IoT service, etc.)
 
+Note: if the `strategy` portion of the `itemlist` is omitted, the `default` strategy specified in the `Strategies` section will be applied.
```

Comment:
Please add an example below.

Reply:
How about the following:

For example, given an Item named `GF_Hall_Light`, if `default` in the `Strategies` section is set to `everyChange`, then the following `Items` section,
```java
Items {
    GF_Hall_Light
}
```
will cause the state of `GF_Hall_Light` to be persisted on every change.